language/sygus.tex

```diff
@@ -2409,6 +2414,19 @@ \subsection{Weights}%
 \end{lstlisting}
 declares a weight keyword $\texttt{numOps}$ whose default weight value is $1$.
 
+\subsection{Oracle communication}
+\label{sec:attr-oracles}
+\begin{enumerate}
+\item The attribute $\texttt{:file}$
```

Comment:
For completeness, can we also add `:command-line` here?  Even though its the default, it would be nice to give syntax for this.

Reply:
added